### ISSUE TITLE
DV | Update intro page content

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#116035]
## Issue Description

While auditing the Dependents Verification Form 21-0538, I noticed that the intro page content doesn't match in these areas:

| Description | Design | Implementation |
|---|---|---|
| Title | <img width="705" height="499" alt="current title content" src="https://github.com/user-attachments/assets/fb48dfc0-9b3f-4b96-8b73-3c707cd5ea8a" /> | <img width="695" height="416" alt="design title content" src="https://github.com/user-attachments/assets/5a0d8b46-7a9d-4535-a8c0-ce73ca3d1c52" /> |
| Alert order | <img width="618" height="539" alt="alert shown below 'follow the steps below' h2" src="https://github.com/user-attachments/assets/d550285b-11ee-4902-a1ea-72eca50acb07" /> | <img width="697" height="484" alt="alert above VA intro text" src="https://github.com/user-attachments/assets/30447dc6-da5c-4d56-af14-4b3dd68dc533" /> |
| Step 4 | N/A (it's incorrect) | <img width="684" height="196" alt="Image" src="https://github.com/user-attachments/assets/bb730898-83a2-414c-830a-4188af356b21" /> |
| Need help | <img width="838" height="168" alt="Image" src="https://github.com/user-attachments/assets/5331b51c-9763-4dcc-8beb-eb2b7d845c08" /> | <img width="707" height="159" alt="design need help content" src="https://github.com/user-attachments/assets/0d4c7683-2927-4f29-ba85-eea2ca31c9fa" /> |

---
## Tasks
- [x] Update H1 to match breadcrumb & design (don't modify the page subtitle)
- [x] Move No dependents alert (Gateway component) below the page subtitle
- [x] The "Use our online tool..." paragraph needs a "va-introtext" class name applied 
- [x] Remove H3 resize on the H2 "Follow the steps below to get started:", and reduce top margin (the H2 class name is missing an "s" in `vad-u-margin-top--0`)
- [x] In step 4, change the single quote in "dependents' information" to a typographical apostrophe
- [x] Update need help component content
- [x] Update unit tests as needed

## Acceptance Criteria
- [x] Intro page matches design
- [x] All tests passing

**Comment by fiorella-io** [comment]
Also:
- [ ] Update font style to `vads-u-line-height--6 vads-u-font-size--h3 vads-u-font-family--serif` for the 1st paragraph "Use our online tool to submit..."
- [ ] Update second heading to `Follow these steps to get started` and use H2 styling

**Staging:**
<img width="50%"  src="https://github.com/user-attachments/assets/3670e86a-04bf-47c5-9424-27ef33841802" />

**Design:**
<img width="50%" src="https://github.com/user-attachments/assets/3b0a54c9-4376-44f1-b05f-b12e3d8e128e" />

**Comment by davidakennedy** [comment]
Also, adding this, @Midge-dev:

Content:

Design says: Here’s what you’ll need to review:

Staging says: Here’s what you will need to review:

Should have "you'll" contraction.

[Figma file](https://www.figma.com/design/bvj72inycD0iZkuCbjYTWL/Dependent-Verification-MVP?node-id=9070-18394&t=AIgQ7xpOB9D7wZMo-4)

Staging:

<img width="686" height="864" alt="Image" src="https://github.com/user-attachments/assets/05fe51f6-5bd6-48b0-9978-d4dbb923f0cc" />

Design:

<img width="360" height="396" alt="Image" src="https://github.com/user-attachments/assets/868fb80e-6638-4830-a87d-1abc6dfb6fdb" />

**Comment by fiorella-io** [comment]
I also just noticed that we need to add spacing between the intro paragraph and "Follow the steps..."

<img width="60%" src="https://github.com/user-attachments/assets/13f09171-60cc-4ce0-a992-9b1e4aefd718" />

**Comment by fiorella-io** [comment]
Looks good! ✅
- Intro page updates
     - <img width="30%"  src="https://github.com/user-attachments/assets/c3fa5530-19c0-4710-947f-37df2060de67" />
- In-progress alert
     - <img width="30%" src="https://github.com/user-attachments/assets/28f28b3a-99c3-4c24-a271-7b2f17c9de5b" />
